crates/wit-bindgen/src/lib.rs

```diff
@@ -465,18 +465,40 @@ impl Wasmtime {
 
                 let name = resolve.name_world_key(name);
                 let (path, method_name) = match pkgname {
-                    Some(pkgname) => (
-                        format!(
+                    Some(pkgname) => {
+                        let mut method_name = format!(
+                            "{}_{}_",
+                            pkgname.namespace.to_snake_case(),
+                            pkgname.name.to_snake_case()
+                        );
+
+                        let version_count = resolve
+                            .package_names
+                            .keys()
+                            .filter(|x| x.name == pkgname.name && x.namespace == pkgname.namespace)
+                            .count();
```

Comment:
I don't think this is exactly right. Was having a hard time mapping interfaces to packages in an effective way but this proved to be a way to get it to build.

looking for betters ways...

Reply:
Coudl this perhaps go through [`name_package_module`](https://github.com/bytecodealliance/wasmtime/blob/62c143e1859945a888fe28d8534ffe0b3d022cc7/crates/wit-bindgen/src/lib.rs#L209-L252) to leverage the preexisting logic to drop the version?

Reply:
That was exactly what I needed and it was right in front of me 😆